packages/synapse-react-client/src/components/widgets/facet-nav/PlotsContainer.tsx

```diff
@@ -89,6 +90,7 @@ const getCombinedNewPlots = (
     FacetNavPanelProps,
     'applyChangesToFacetFilter' | 'applyChangesToGraphSlice' | 'facetToPlot'
   >[] = [],
+  plotType: PlotType,
```

Comment:
Prefer to put the default in the argument definition instead of `??` operator if it's cleaner
```suggestion
  plotType: PlotType = DEFAULT_PLOT_TYPE,
```